htdocs/core/lib/pdf.lib.php

```diff
@@ -2618,8 +2618,9 @@ function pdf_getLinkedObjects(&$object, $outputlangs)
 	if (is_object($hookmanager)) {
 		$parameters = array('linkedobjects' => $linkedobjects, 'outputlangs' => $outputlangs);
 		$action = '';
-		$hookmanager->executeHooks('pdf_getLinkedObjects', $parameters, $object, $action); // Note that $action and $object may have been modified by some hooks
-		if (!empty($hookmanager->resArray)) {
+		$reshook = $hookmanager->executeHooks('pdf_getLinkedObjects', $parameters, $object, $action); // Note that $action and $object may have been modified by some hooks
+		if ($reshook>0) {
```

Comment:
If reshook is 0, it means the hook ask to concat, then the resArray must be concatenate to linkedobjects.
If reshook is 1, it means the hook asked to replace completely.
so i think we are missing the cas reshook == 0 that was correctly supported before.